docs/application/native/guides/ui/efl/learn-edc-part.md

```diff
@@ -701,17 +701,17 @@ part {
 
     Sets the minimum size calculation. See the `edje_object_size_min_calc()` and `edje_object_size_min_restricted_calc()` functions of the Edje Object Geometry API (in [mobile](../../../api/mobile/latest/group__Edje__Object__Geometry__Group.html) and [wearable](../../../api/wearable/latest/group__Edje__Object__Geometry__Group.html) applications). The value tells the min size calculation routine where this part does not change size in width or height (1 for it does not, 0 for it does), so the routine does not try to expand or contract the part.
```

Comment:
Suggest a rephrase, please check if the below sentence is technically accurate?

The value tells the minimum size calculation routine where this part does not change size in width or height, so the routine does not try to expand or contract the part. The value 1 indicates it does not change size, and the value 0 denotes there is size change.


min in code tag or change to minimum